pkg/driver/controller.go

```diff
@@ -76,6 +76,9 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	if err := d.isValidVolumeCapabilities(volCaps); err != nil {
 		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("Volume capabilities not supported: %s", err))
 	}
+	if err := d.validateFStype(volCaps); err != nil {
```

Comment:
realizing now this would be a breaking change. Maybe it is sufficient to remove the check from nodePublish and simply ignore fsType in both nodePublish and createVolume paths

Reply:
even in our example storageclasses we neglect to set fsType. So i think we need to tolerate case where fsType is nonsense, unset, or ext4 :/

Reply:
do you mean we remove the check for both nodePublish and createVolume?

Reply:
yeah. sorry for lack of clarity, i am not 100% sure. The likelihood of someone having an EFS storageclass with nonsense fsType is very small.

Reply:
let's keep it. As loong as there is a release note we should be OK